internal/common/disasters/environmentregistration.go

```diff
@@ -3,13 +3,13 @@ package disasters
 import "github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 
 // InitEnvironment initialises environment according to definitions
-func InitEnvironment(islandIDs []shared.ClientID, xBounds [2]float64, yBounds [2]float64, disasterParams DisasterParameters) (*Environment, error) {
+func InitEnvironment(islandIDs []shared.ClientID, xBounds [2]float64, yBounds [2]float64, disasterParams DisasterParameters, cp Commonpool) (*Environment, error) {
 
 	ag := ArchipelagoGeography{map[shared.ClientID]Island{}, xBounds, yBounds}
 
 	for i, id := range islandIDs {
 		island := Island{id, float64(i), float64(0)} // begin with points on x axis
 		ag.islands[id] = island
 	}
-	return &Environment{ag, disasterParams, DisasterReport{}}, nil // may want ability to return error in future
+	return &Environment{ag, disasterParams, DisasterReport{}, cp}, nil // may want ability to return error in future
```

Comment:
See example in coding rule 1: 
https://github.com/SOMAS2020/SOMAS2020/blob/main/docs/SETUP.md#coding-rules

Name the arguments of the struct